ext/readline/readline.c

```diff
@@ -307,6 +315,23 @@ PHP_FUNCTION(readline_info)
 			RETVAL_STRING(SAFE_STRING(rl_prompt));
 		} else if (!strcasecmp(what, "terminal_name")) {
 			RETVAL_STRING((char *)SAFE_STRING(rl_terminal_name));
+		} else if (!strcasecmp(what, "completion_suppress_append")) {
+			oldval = rl_completion_suppress_append;
+			if (value) {
+				convert_to_long_ex(value);
+				rl_completion_suppress_append = Z_LVAL_P(value);
+			}
+			RETVAL_LONG(oldval);
+		} else if (!strcasecmp(what, "completion_append_character")) {
+			char completion_append_str[2];
+			oldval = rl_completion_append_character;
+			if (value) {
+				convert_to_string_ex(value)
+				rl_completion_append_character = (int)strdup(Z_STRVAL_P(value))[0];
```

Comment:
This strdup is going to leak. This code should just be `(int) Z_STRVAL_P(value)[0]`.

Reply:
Ah I see, thank you